### ISSUE TITLE
chains start --init-dir requires the chain name to match the directory & CONFIG_PATH bug

### ISSUE THREAD

**Issue by j-h-scheufen** [eris-ltd/eris-cli#964]
This was experienced after an `eris update --branch=develop`.

These commands don't work:
```
$> eris chains make test3 --chain-type=simplechain
$> eris chains start test2 --init-dir ~/.eris/chains/test3/
The chain directory provided does not exist, re-run with an existing directory
```

It only works when naming the chain as `test3`, so it matches the `~/.eris/chains/test3` directory.
This behavior is regressive as it was possible with the former version to decide the name of the chain independent of the directory created from `eris make`.
Users need to be able to instantiate the same chain configuration multiple times under different names.

OSX version: 10.11.6
Eris CLI Version: 0.12.0-rc3
Docker:
Client:
 Version:      1.11.1
 API version:  1.23
 Go version:   go1.5.4
 Git commit:   5604cbe
 Built:        Tue Apr 26 23:44:17 2016
 OS/Arch:      darwin/amd64

Server:
 Version:      1.12.1
 API version:  1.24
 Go version:   go1.6.3
 Git commit:   23cf638
 Built:        Thu Aug 18 17:52:38 2016
 OS/Arch:      linux/amd64
docker-machine version 0.7.0, build a650a40

**Comment by zramsay** [comment]
related (ish) to #858. we can revert to the previous state - I added the check as an over-simplification -  > Users need to be able to instantiate the same chain configuration multiple times under different names
Interesting, i've always tended towards `chains make`ing a new chain - flexibility in the name/dir makes sense though.

**Comment by zramsay** [comment]
Note: in this version, you can drop the `--chain-type=simplechain` - it's assumed if no flag is given. Also, `eris update` is gone

**Comment by zramsay** [comment]
This will require a re-work of how cli finds the config file. Right now, chainName is assumed to == a chains main directory. cli also looks in the named chains dir to find CONFIG_PATH _then_ gets the config file. This is overcomplicated and overopiniated.

Proposed solution: one file (`~/.eris/chains/configs.csv`) with `chainName,path/to/config.toml` as each line. Any chains command start there to `LoadChainDefinition` from `config.toml`

**Comment by zramsay** [comment]
closing per today's discussion. The config path issue is being resolved as part of #991 and #994 

For the time being, and due to the refactor, it won't be possible to `make` a chain then `start` it with a different name. There are some underlying assumptions in cli and edb that prevent this (to be solved in #858 and #336 )

**Comment by pietv** [comment]
@j-h-scheufen the fix implemented for the #991 in the current #develop branch doesn't require the chain name to match the path for the `--init-dir` directory.
